dask/sizeof.py

```diff
@@ -84,6 +84,13 @@ def sizeof_pandas_index(i):
             p += object_size(i)
         return int(p) + 1000
 
+    @sizeof.register(pd.MultiIndex)
+    def sizeof_pandas_multiindex(i):
+        p = int(sum(object_size(l) for l in i.levels))
+        for c in (i.codes if hasattr(i, 'codes') else i.labels):
+            p += c.nbytes
```

Comment:
What is this for?  Categorical indexes?  If so, should there be a test?  Also should we do something like call `sizeof(i.codes)` instead of the for loop here?

Reply:
The linked issue is specifically about `MultiIndex` but `CategoricalIndex` presumably has the same issue. The added test case uses a multiindex but it's a bit unclear so I just made it a separate test.

`codes` is a `FrozenList` of `FrozenNDArray`s; each one has an `nbytes` but the list itself does not. Since this is only looping over the levels of the index it seems fine to me.

Reply:
OK.  Thanks for the explanation.